### ISSUE TITLE
Linked filters: if a selection in one filter imply a unique value in another, set the filter variable to that value 

### ISSUE THREAD

**Issue by e-gaulue** [metabase/metabase#29203]
**Is your feature request related to a problem? Please describe.**
Well, linked filters look really frustrating. Lots of issue here and question on discourse. We seem to discuss with an IA: we know what we want and we try to do everything for this IA to understand our need. It goes through the questions, then sub-questions then in the  model to guess the relationships, but sadly, it looks it happens too often it doesn't manage.

**Describe the solution you'd like**
We know the relationship between our filters. We can sum them up to a table with: Country, District, City or Grand Parents, Parents, Children. Of course, the system can guess this table from the model, but it looks it is not so easy.
As far as we have to build the questions that will be used by our cards on the dashboard, it is not hard for us to build this table (even if we don't show all the fields). When we link the filters to the cards, we know who is the grand-parent, the parent or the child, and in a certain way we apply the PKs and FKs.

It would be great to propose to select a dashboard card on which we should use the underlying datas to filter the filters. In the linked filter tab, when we activate the filter which is linked, we could have a radio button that appears with:
Data to use to filter the linked filters:
:radio_button: Infered from the model (would be the default, so everything stay the same)
:white_circle: First card where both filters are linked
:white_circle: Second card where both filters are linked
:white_circle: Third card where both filters are linked
:white_circle: ...

This way, all current linked filters would go on working the same as 'Infered from the model' would be the default. But anyone could provide a table (question) to use when the situation is not so easy to guess.

**Describe alternatives you've considered**
Spend several hours to understand how metabase try to guess the relationship without success, reading https://www.metabase.com/docs/latest/troubleshooting-guide/linked-filters and more. Wondering what was necessary (PK, FK in the model) and optional (model type).
I finished with a PHP script I use as a proxy that knowing one of parameter set the other ones (exemple: knowing the parent, I know the grand parent), so I can set those as parameters in the Metabase dashboard URL. Maybe linked with #28404.

**How important is this feature to you?**
Well of course it's important for me, but maybe for all of us. I think it's time consuming to answer so much questions/help regarding these linked filters, so maybe it deserves it. Once agin, most of the time, dashboard designers know exactly how they would like those filters to work.

**Comment by e-gaulue** [comment]
**Warnings**
I wonder if my request is not a misunderstanding on what are linked filters.

I have a more restrictive case than Country, District, City. I've got a table with Contacts (n records) and a table with Customers (m records), where n>m and when I've got a relationship it's a 1<->1.

My dashboard can be use knowing the ContactId or CustomerId, those correspond to two filters on their respective "Id". What I'm looking for is:
- if CustomerId is used and exists, then ContactId should be update (each Customer is a Contact)
- if ContactId is used and exists, then if I've got a corresponding Customer: update CustomerId filter else empty it

I though it was a feature of linked filters. But maybe my settings are goods, and it's just I observe #28401.
